iree/test/e2e/models/mobilenetv2_fake_weights.mlir

```diff
@@ -986,7 +987,8 @@ module  {
     }) {dimensions = dense<1> : tensor<1xi64>} : (tensor<1x1000xf32>, tensor<f32>) -> tensor<1xf32>
     %708 = "mhlo.broadcast_in_dim"(%707) {broadcast_dimensions = dense<0> : tensor<1xi64>} : (tensor<1xf32>) -> tensor<1x1000xf32>
     %709 = mhlo.divide %706, %708 : tensor<1x1000xf32>
-    return %709 : tensor<1x1000xf32>
+    check.expect_almost_eq_const(%709, dense<0.001> : tensor<1x1000xf32>) : tensor<1x1000xf32>
```

Comment:
I am not sure how stable such a small value will be. But Looks fine to me.

Reply:
MobileNet is classifying the input image and output a probability for all 1000 categories. As long as we give a splat value as input (which means the input image is really just containing pixels of the same RGB values), the output would be low probability for all categories since the input image does not has any meaningful objects. I think we don't want to go the extra steps to put in real images here given this is just fake weight. We have integration tests for that purpose.

Reply:
Yeah. Thats not what I meant sorry. The absolute value here is getting closer to fp32 round of errors. So it "can" cause issues. If we just bumped the fake weights values up, and that makes this higher, maybe. Just a caution. Thats why I didnt block it.